deps/SnoopCompile/precompile/precompile_BinaryBuilder.jl

```diff
@@ -1,205 +0,0 @@
-const __bodyfunction__ = Dict{Method,Any}()
```

Comment:
:scream:

Reply:
We can add it back in the future, but yeah, it was too out of date.  :P